src/pyedb/dotnet/edb_core/configuration.py

```diff
@@ -678,3 +682,33 @@ def _load_padstacks(self):
                         backdrill_bottom["drill_diameter"],
                         backdrill_bottom["stub_length"],
                     )
+
+    @pyedb_function_handler
+    def _load_package_def(self):
+        """Imports package definition information from JSON."""
+        comps = self._pedb.components.components
+        for pkgd in self.data["package_definitions"]:
+            name = pkgd["name"]
+            if name in self._pedb.definitions.package:
+                self._pedb.definitions.package[name].delete()
+            from pyedb.dotnet.edb_core.definition.package_def import PackageDef
```

Comment:
Is it possible to move that call at the top of the module or does it creates circular import ?

Reply:
No. I will correct this.